.github/workflows/blog.yaml

```diff
@@ -46,7 +46,11 @@ jobs:
             ${{ runner.os }}-blogdown2-${{ hashFiles('environment.yml') }}-
             ${{ runner.os }}-blogdown2-
       - name: Build site
+        env:
+          API_KEY: ${{ secrets.DELPHI_GITHUB_ACTIONS_EPIDATA_API_KEY }}
         run: |
+          printenv
+          echo $API_KEY
```

Comment:
```suggestion
```

Side-note, this is a dangerous way to test, since CI output is publicly visible, which sorta defeats the purpose of having a secret.

Reply:
it's okie github automatically hides secrete when print out things. so you see a bunch of **** in the log instead of the actual secret content.